### PR TITLE
sbt 1.6 (was 1.5)

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

**Comment by SethTisue** [comment]
I'll return to this.

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.5.5
+sbt.version=1.6.0
```
Comment (patriknw) [inline project/build.properties]: 1.6.1 ?

Comment (SethTisue) [reply project/build.properties]: done

**Comment by SethTisue** [comment]
CI is now failing because our Fortify license expired. I'll work on getting a new one.

**Pull request merged**